docker/ubuntu/Dockerfile

```diff
@@ -1,5 +1,15 @@
-FROM ubuntu:20.04
+FROM ubuntu:20.04 as base
+ENV DEBIAN_FRONTEND=noninteractive
+
 RUN apt-get update && \
-    apt-get install -y python3-pip wget libffi-dev git jq rubygems apt-transport-https && \
-    pip3 install boto awscli && \
+    apt-get install -y python-is-python3 python3-pip wget libffi-dev git jq rubygems apt-transport-https && \
+    gem install fpm && \
+    rm -rf /var/lib/apt/lists/*
+    # TODO: maybe move build deps such as fpm away from here
+
+
+
+FROM base as upload
```

Comment:
This had to be split, since `boto3` installed with pip (`dvc` dependency) conflicts with `boto`